docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
+that can be packaged in a container image layer , a `git` repository or any other content source that the
+[plain bundle provisioner](../provisioners/plain.md) supports.
+
+Supported source types for a plain bundle currently include the following:
+
+* A directory in a [container image](../sources/image.md)
+* A directory in a [`git` repository](../sources/git.md)
+* A set of keys in a [`ConfigMap`](../sources/local.md)
+* An [upload](../sources/upload.md)
+* A `.tgz` file returned by a [http endpoint](../sources/http.md)
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
 
 The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
-combines the type of bundle (plain) with the current schema version (v0).
+combines the type of bundle (plain) with the current schema version (v0). The
+[plain bundle provisioner](../provisioners/plain.md) is able to source
+`plain+v0` bundles and install them onto a Kubernetes cluster.
 
-> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> Note: the `plain+v0` bundle format is at schema version v0, which means it's an experimental format that is subject
 > to change.
 
-## Example
+## Common Terminology
+
+* *bundle* is a collection of files that define content to be deployed to a cluster
+* *bundle image* is a container image that contains a bundle within its filesystem
+* *bundle git repo* is a git repository that contains a bundle within a directory
+
+## Technical Details
+
+* The static manifests must be located in the root-level `/manifests` directory for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack.
+* The manifests directory should be flat: all manifests should be at the top-level with no subdirectories.
+* It is required that `kubectl apply` is able to process all .yaml files in the directory that make up a plain bundle. For example, multi-object YAML files are acceptable, but Ansible playbooks would not be.
+
+## Building a plain bundle
+### Prerequisites
 
-For example, below is an example of a file tree in a `plain+v0` bundle. It must have a `manifests` directory contains
-the Kubernetes resources required to deploy an application.
+In order to create a plain bundle for Rukpak, ensure your Kubernetes manifests are in a flat directory at the root of
```

Comment:
there are multiple ways of writing RukPak in this document. Can we use "RukPak" everywhere?

Reply:
Thanks for pointing this out! It should be standardized as of https://github.com/operator-framework/rukpak/pull/642/commits/902da450e601fa3c1b012ca90b21745fb1ecaa25 - let me know if you notice any spots that I missed!